### DISCUSSION TITLE
Is there a release schedule?

### DISCUSSION THREAD

**Discussion by Dzieni** [nodejs/undici#2207] [General]
I've been wondering how do you decide to release a new version?

I tried to look at the history to find some patterns and I don't feel there is any:

- there was 2 minor and 2 patch releases in January
- 4 minor and 2 patch releases in February (great job!)
- 1 minor release in March
- 1 minor and 2 patch releases in April
- and a 1 patch release in May
- no release in June and July

I'm asking because yesterday at @StratoKit we spontaneously encountered an issue #2171 and started fixing it, just to realize that a fix has been on the main branch since the end of June. We'd love to just use that fix without having to point to a specific commit when installing Undici.

So, is there a plan for 5.23.0 or 5.22.2 in a forseeable future?

**Comment by ronag** [comment]
We don't have any release schedule. But I've published https://github.com/nodejs/undici/releases/tag/v5.23.0 for you now.

**Reply by Dzieni** [reply]
Thank you very much!
